mdwa_planning/Barrier[] barrier
mdwa_planning/Dy_Barrier[] dy_obs
float32 obs_size
